tests/providers/test_address.py

```diff
@@ -526,6 +526,14 @@ def test_state(self):
         state = self.factory.state()
         assert isinstance(state, string_types)
 
+    def test_street_title(self):
+        street_title = self.factory.street_title()
+        assert isinstance(street_title, string_types)
+
+    def test_street_suffix(self):
+        suffix = self.factory.street_suffix()
+        assert isinstance(suffix, string_types)
+        assert suffix in FiProvider.street_suffixes
```

Comment:
`FiProvider` is a Finnish provider. I don't think you want to test against that one. You'll have to import the Hindi provider (if it's not imported already) and check against that.